src/components/Calculator.js

```diff
@@ -1,30 +1,37 @@
 function Calculator() {
   return (
```

Comment:
Please kindly implement at least one child component for the Calculator component. This [tutorial](https://www.freecodecamp.org/news/pass-data-between-components-in-react/) is self explanatory and might be of help.